apps/dms-material/src/app/global/global-universe/services/universe.service.ts

```
@@ -0,0 +1,56 @@
+          this.cachedUniverses.set(universesArray);
+        }
+      }
+    );
+  }
+
+  /**
+   * Computed signal that provides universes from SmartNgRX store.
+   * Uses cached data when SmartNgRX temporarily returns empty (during state recalculation).
+   */
+  // eslint-disable-next-line @smarttools/no-anonymous-functions -- Arrow function required for proper 'this' binding in computed signal
+  universes = computed(() => {
```

ESLint

[error] Refactor this function to not always return the same value.

(sonarjs/no-invariant-returns)